src/main.rs

```diff
@@ -58,6 +58,7 @@ struct Board {
     height: isize,
     width: isize,
     food: Vec<Coord>,
+    hazards: Vec<Coord>,
```

Comment:
does this get passed in in other modes? Might need to make this optional

Reply:
Not optional according to the api spec
https://docs.battlesnake.com/references/api#board